guides/source/action_view_helpers.md

```diff
@@ -507,166 +717,328 @@ number_with_precision(111.2345)               # => 111.235
+marker will be appended to the result for a total length not exceeding `length`.
 
 ```ruby
-url_for @profile
-# => /profiles/1
+truncate("Once upon a time in a world far far away")
+# => "Once upon a time in a world..."
 
-url_for [ @hotel, @booking, page: 2, line: 3 ]
-# => /hotels/1/bookings/1?line=3&page=2
+truncate("Once upon a time in a world far far away", length: 17)
+# => "Once upon a ti..."
 
-url_for @post # given a composite primary key [:blog_id, :id]
-# => /posts/1_2
+truncate("Once upon a time in a world far far away", length: 17, separator: ' ')
+# => "Once upon a..."
+
+truncate("And they found that many people were sleeping better.", length: 25, omission: '... (continued)')
+# => "And they f... (continued)"
+
+truncate("<p>Once upon a time in a world far far away</p>", escape: false)
+# => "<p>Once upon a time in a wo..."
 ```
 
-#### link_to
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-truncate)
+for more information.
+
+#### word_wrap
+
+Wraps the text into lines no longer than `line_width` width.
+
+```ruby
+word_wrap('Once upon a time', line_width: 8)
+# => "Once\nupon a\ntime"
+```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-word_wrap)
+for more information.
+
+### TagHelper
+
+Provides methods to generate HTML tags programmatically.
+
+#### content_tag
+
+Generates an HTML block tag of the specified type to surround the content. You
+can add HTML attributes by passing an `attributes` hash to options.
+
+```ruby
+content_tag(:div, "Hello world!", class: ["strong", { highlight: current_user.admin? }])
+# => <div class="strong highlight">Hello world!</div>
+ ```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag)
+for more information.
+
+#### tag
+
+Generates a standalone HTML tag with the given `name` and `options`.
+
+```ruby
+tag.h1 'All titles fit to print'
+# => <h1>All titles fit to print</h1>
+
+tag.section class: %w( kitties puppies )
+# => <section class="kitties puppies"></section>
+```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-tag)
+for more information.
+
+### UrlHelper
+
+Provides methods to make links and get URLs that depend on the routing
+subsystem.
+
+#### button_to
+
+Generates a form that submits to the passed URL. The form has a submit button
+with the value of the `name`.
+
+```html+erb
+<%= button_to "Sign in", sign_in_path %>
+```
+
+would output the following HTML:
+
+```html
+<form method="post" action="/sessions" class="button_to">
+  <input type="submit" value="Sign in" />
+</form>
+```
+
+See the [API
+Documentation](https://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-button_to)
+for more information.
+
+#### current_page?
+
+Returns true if the current request URI matches the given `options`.
```

Comment:
As the examples above use URL instead URI (and it's for the UrlHelper), I think we should URL here as well:
```suggestion
Returns true if the current request URL matches the given `options`.
```